client/task-list/test/index.js

```diff
@@ -135,6 +135,24 @@ describe( 'TaskDashboard and TaskList', () => {
 		).toBeNull();
 	} );
 
+	it( 'renders only the extended task list', () => {
```

Comment:
Could we also add a test for when the query object has `task` defined?

Reply:
Good idea! I added the test in the commit b62f9cf

Reply:
Thanks @octaedro, looks great :)